console/console-init/ui/src/modules/device/dialogs/CreateDevice/CreateDevice.tsx

```diff
@@ -78,6 +80,16 @@ const CreateDevice: React.FunctionComponent<IDeviceInfo> = ({
           onConnectionChange={onConnectionChange}
         />
       )
+    },
+    {
+      name: "Add Gateway",
+      component: (
+        <AddGateway
+          addedGateways={[]}
+          removeGateway={removeGateway}
+          addGateway={(id: string) => {}}
```

Comment:
create a handler like removeGateway. Also try to implement the functioanlity to add and remove gateway by declaring a addedgateway as a  state for now. Also in Add gateway function make a comment with `TODO: verify the device Id with server`

Reply:
done